src/ReturnTypes/AppMakeHelper.php

```diff
@@ -41,6 +43,11 @@ public function resolveTypeFromCall(FuncCall|MethodCall|StaticCall $call, Scope
                     $resolved = $this->resolve($constantString->getValue());
 
                     if ($resolved === null) {
+                        if ($constantString->isClassString()) {
```

Comment:
sorry, this returns a TrinaryLogic instead of a boolean

```suggestion
                        if ($constantString->isClassString()->yes()) {
```

Reply:
Thanks hadn't had time to really look at ityet.